apps/laboratory/src/utils/SiweUtils.ts

```diff
@@ -4,17 +4,17 @@ import type { SIWEVerifyMessageArgs, SIWECreateMessageArgs, SIWESession } from '
 import { createSIWEConfig } from '@web3modal/siwe'
 
 export const siweConfig = createSIWEConfig({
-  createMessage: ({ nonce, address, chainId }: SIWECreateMessageArgs) =>
-    new SiweMessage({
-      version: '1',
-      domain: window.location.host,
-      uri: window.location.origin,
-      address,
-      chainId,
-      nonce,
-      // Human-readable ASCII assertion that the user will sign, and it must not contain `\n`.
-      statement: 'Sign in With Ethereum.'
-    }).prepareMessage(),
+  messageParams: {
+    domain: 'lab.web3modal.com',
+    uri: 'https://lab.web3modal.com',
```

Comment:
Why is it hardcoded ?
```suggestion
     domain: window.location.host,
     uri: window.location.origin,
```